ResearchUI/ResearchUI/iOS/Views/RSDStepChoiceCell.swift

```diff
@@ -344,23 +344,162 @@ open class RSDStepTextFieldCell: RSDTableViewCell {
     }
 }
 
+/// `RSDStepTextViewCell` is the base implementation of a text view used to enter answers in a
+/// form step table view.
+open class RSDStepTextViewCell: RSDTableViewCell {
+    
+    /// The text field associated with this cell.
+    public var textView: RSDStepTextView!
+    
+    /// The label used to display the prompt for the input field.
+    open var viewLabel: UILabel!
+    
+    /// Layout constants. Subclasses can override to customize; otherwise the default private
```

Comment:
They aren't actually private though, are they? Just not writable outside the class itself?